portfolio/src/main/webapp/index.html

```diff
@@ -292,6 +296,7 @@ <h5 class="blue-text">
           </select>
           <input id="content" type="text" 
               placeholder="Your Recommendation?" name="content">
+          <input type="file" id="image" name="image" placeholder="upload image">
```

Comment:
I don't think placeholder works for input type file, feel free to remove it. I think actually changing the default text would require more work

Reply:
Ah yes, you are right!